### ISSUE TITLE
ERD Complete & Approved

### ISSUE THREAD

**Issue by taylorseager** [taylorseager/CulinaryChronicles#2]
# User Story
The ERD will outline the data keys and the values they should have. This will be updated when I hit MVP with stretch goal data. 

# Acceptance Criteria
Recipe Table
-title (string)
-firebasekey (string)
-categoryId (string)
-description (text)
-ingredients (string)
-author (string)
-totalTime (num)
-image (text)
-favorite (bool)
-uid (string)

Category Table (all strings)
-type 
-uid
-firebaseKey
-image

# Dependecies
This should be completed first to ensure the dataset is correct.

# Dev Notes
Needs to be approved in planning prior to any coding

![Screenshot 2024-05-14 at 6 31 23 PM](https://github.com/taylorseager/CulinaryChronicles/assets/154686027/13fe5255-6271-49e6-98b6-d447d8168cd6)
